From e81dcdfef48c5212fce4d504139095bd0ef0c8fd Mon Sep 17 00:00:00 2001
From: Jagannadha Sutradharudu Teki <jagannadha.sutradharudu-teki@xilinx.com>
Date: Thu, 28 Mar 2013 02:49:07 -0700
Subject: [PATCH] xilinx: dts: Update the memory amount for zc706

Updated the actual memory amount used by zc706 board.

This changes is follow-up on memory amount in previous patch.
"Xilinx: ARM: Added zc706 device tree and compatibility"
(sha1: 082f42fd30e07bb35100c435199f56adef932875)

Signed-off-by: Jagannadha Sutradharudu Teki <jaganna@xilinx.com>
---
 arch/arm/boot/dts/zynq-zc706.dts |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/zynq-zc706.dts b/arch/arm/boot/dts/zynq-zc706.dts
index ed8cd39..ef1e1b6 100644
--- a/arch/arm/boot/dts/zynq-zc706.dts
+++ b/arch/arm/boot/dts/zynq-zc706.dts
@@ -9,7 +9,7 @@
 
 	memory {
 		device_type = "memory";
-		reg = <0x00000000 0x10000000>;
+		reg = <0x00000000 0x40000000>;
 	};
 	chosen {
 		bootargs = "console=ttyPS0,115200 root=/dev/ram rw ip=:::::eth0:dhcp earlyprintk";
-- 
1.7.1

